### PR TITLE
feat(parse-srcset): new package parse-srcset types added

### PR DESCRIPTION
**Pull request by dylanarmstrong**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/parse-srcset/parse-srcset-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d7549b2518e67cff8c5ea86908a2ef02ef5fec7b
</p>

File: types/parse-srcset/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+interface ParsedSrcset {
+    url: string;
+    d?: number;
+    h?: number;
+    w?: number;
+}
+
+declare function parseSrcset(value: string): ParsedSrcset[];
+
+export = parseSrcset;
```
Comment (jakebailey) [inline types/parse-srcset/index.d.ts:10]: This is a UMD module, you probably also want `export as namespace parseSrcset`

Comment (dylanarmstrong) [reply types/parse-srcset/index.d.ts:11]: Thanks, good catch, I've updated.

**Review by jakebailey — APPROVED** [review]

**Comment by dylanarmstrong** [comment]
Ready to merge

**Pull request merged**